tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -92,7 +95,9 @@ public function testPostCreate() {
 		$imageHeight = 100;
 		$dimensions = array($imageWidth, $imageHeight);
 
-		$this->mockGetImageSize($this->imagehooks, $location, $dimensions);
+		$this->mockImageHooks = $this->mockPostCreate(['postCreate',]);
```

Comment:
You'll know because you're checking that `imageDimension` methods are called once, using the image dimension as arguments.
You see what I mean?